OpenOrchestrator/orchestrator/tabs/queue_tab.py

```diff
@@ -70,17 +71,19 @@ def __init__(self, queue_name) -> None:
 
         with ui.dialog(value=True).props('full-width full-height') as dialog, ui.card():
             with ui.row().classes("w-full"):
-                self._create_column_filter()
+                self.from_input = DatetimeInput("From Date", on_change=self._update, allow_empty=True).style('margin-left: 20px')
```

Comment:
Why do you define a margin?

Reply:
I can see in the css that the padding of a scroll area is 1rem (16px). If you set it to that it should also scale correctly.

```suggestion
                self.from_input = DatetimeInput("From Date", on_change=self._update, allow_empty=True).style('margin-left: 1rem')
```